db/db_impl.cc

```diff
@@ -2737,6 +2739,54 @@ Status DBImpl::IngestExternalFile(
   return status;
 }
 
+Status DBImpl::VerifyChecksum() {
+  Status s;
+  Options options;
+  EnvOptions env_options;
+  std::vector<ColumnFamilyData*> cfd_list;
+  {
+    InstrumentedMutexLock l(&mutex_);
+    for (auto cfd : *versions_->GetColumnFamilySet()) {
+      if (!cfd->IsDropped() && cfd->initialized()) {
+        cfd->Ref();
+        cfd_list.push_back(cfd);
+      }
+    }
+  }
+  std::vector<SuperVersion*> sv_list;
+  for (auto cfd : cfd_list) {
+    sv_list.push_back(cfd->GetReferencedSuperVersion(&mutex_));
+  }
+  for (auto& sv : sv_list) {
+    VersionStorageInfo* vstorage = sv->current->storage_info();
```

Comment:
`SuperVersion` says:

```
  // Accessing members of this class is not thread-safe and requires external
  // synchronization (ie db mutex held or on write thread).
```

Is it wrong in case of `current`?